### PR TITLE
CRAN release prep

### PR DESCRIPTION
**Pull request by edgararuiz**

### CODE REVIEW

**Comment by edgararuiz** [comment]
@drsimonj - Unless you have really strong feelings about closing #40, I think this gives us a good baseline for a CRAN release.  I'll update the `cran-comments` once we're happy with the PR. 

BTW - I plan to send over improvements to handling `tbl_sql()` object after the next release of `dbplyr`.  That's another reason I'd like to get us on a newer version of the package

**Comment by drsimonj** [comment]
Thanks @edgararuiz, looks great. I agree we can move ahead without tackling #40 

Only thing I noticed was version number. corrr has undergone a lot of dev since last release (including some minor breaking changes), so I'd prefer to increment to 0.3.0 unless you have a reason otherwise?

File: README.md [hunk 1]
```diff
@@ -1,59 +1,85 @@
...
 devtools::install_github("drsimonj/corrr")
 ```
 
-Using corrr
------------
+## Using corrr
 
-Using `corrr` typically starts with `correlate()`, which acts like the base correlation function `cor()`. It differs by defaulting to pairwise deletion, and returning a correlation data frame (`cor_df`) of the following structure:
+Using `corrr` typically starts with `correlate()`, which acts like the
+base correlation function `cor()`. It differs by defaulting to pairwise
```
Comment (drsimonj) [inline README.md]: Why new line?

**Comment by edgararuiz** [comment]
Right, I plan to make the ver bump.  And, it seems that re-rendering the .md made these changes. is that a problem?

**Comment by drsimonj** [comment]
I thought it was unusual but all good if it renders properly.

Otherwise, please merge with ver bump 👍

**Pull request merged**